.github/workflows/deploy-latest.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy latest docker.
+
+on:
+  push:
+    branches:
+      - "master"
+      - "staging"
+
+jobs:
+
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+    steps:
+      - uses: act10ns/slack@v1
+        with:
+          status: starting
+          channel: '#opensource-cicd'
+        if: always()
```

Comment:
From what I understand from the documentation I don't think `if: always()` is necessary in this case. (no previous jobs/steps)